### ISSUE TITLE
[OSX] YACReader quits when last window closes

### ISSUE THREAD

**Issue by MaddTheSane** [YACReader/yacreader#117]
This is contrary to how other Mac OS X apps work.

**Comment by prantlf** [comment]
Yes, behaving similarly to other Mac applications in going full-screen, closing the window, quitting the application (`Ctrl+Cmd+F`, `Cmd+W`, `Cmd+Q`) would help the adoption on the Mac platform. On the other hand, I got used to the behaviour of `Esc` pretty quick :-)
